index.html

```diff
@@ -281,6 +281,11 @@ <h4>Game streaming</h4>
              latency for audio, video and data under varying network conditions. This may
              include tweaking of transport parameters for both media and data.</td>
         </tr>
+	<tr>
+           <td>N36</td>
```

Comment:
How is this different from https://github.com/w3c/webrtc-nv-use-cases/pull/14?

Reply:
> How is this different from #14?

I believe N29 is for creating a permission for the one-way communication. And this requirement is for allowing the normal cloud gaming service when there are no need to have two-way communication without requesting user prompt. Also as @aboba shared during the call, I think it would be important to have an access on the information on the hardware accelerated video decoding which is crucial to support higher resolution and refresh rates for the better gaming experiences.